### PR TITLE
Vol 1. up for review

### PR DESCRIPTION
**Pull request by Shuckdubbie**

### CODE REVIEW

File: src/components/options/ExerciseOptions.js [hunk 1]
```diff
@@ -32,14 +33,15 @@ class ExerciseOptions extends React.Component {
       .then(res => {
         let currentExerciseSite = res.currentExerciseSite || defaultExerciseSite.name;
         let exerciseSites = res.exerciseSites || defaultExerciseSites;
-        let exerciseDuration = res.exerciseDuration || defaultexerciseDuration;
+        let exerciseDuration = defaultexerciseDuration;
```
Comment (mircealungu) [inline src/components/options/ExerciseOptions.js]: why not use the exercise duration from storage anymore?

File: src/components/options/Statistics.js [hunk 2]
```diff
@@ -9,13 +9,45 @@ import {
...
+  }
+
+];
 
 class Statistics extends React.Component {
   state = {
-    interceptsData: [],
-    timeSpentLearningData: []
+    interceptsData: [{"name":"a.dk", "value":2},
+                    {"name":"b.dk", "value":3},
```
Comment (mircealungu) [inline src/components/options/Statistics.js]: hmm. what's this?

File: src/pages/Intercepted.js [hunk 3]
```diff
@@ -34,9 +39,11 @@ class Intercepted extends React.Component {
 
             if (!document.hasFocus()) timePassed = 0;
 
+            let skipTimeLeft = this.state.skipTimeLeft - timePassed;
+
             let timeLeft = this.state.timeLeft - timePassed;
             
-            if (timeLeft <= 0) clearInterval(this.state.timer)
+            //if (timeLeft <= 0) clearInterval(this.state.timer)
```
Comment (mircealungu) [inline src/pages/Intercepted.js:46]: why leave commented lines of code in? 
if it's not necessary, remove it.

File: src/pages/Intercepted.js [hunk 4]
```diff
@@ -92,10 +112,21 @@ class Intercepted extends React.Component {
     }
 
     onContinue() {
+        this.timeout();
+        setHistoricalFirebase('Succes');
```
Comment (mircealungu) [inline src/pages/Intercepted.js:117]: i've seen this function a lot of times, and I still don't 
understand what it does. what is the meaning of "set historical firebase"?
can you rename it in a more intention-revealing way?

File: src/util/fire.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import firebase from 'firebase';
```
Comment (mircealungu) [inline src/util/fire.js]: why not name the file firebase.js? you're not saving that much with calling it "fire"

**Review by mircealungu — CHANGES_REQUESTED** [review]

**Comment by mircealungu** [comment]
@Shuckdub - any progress on answering my feedback?
then we can eventually merge the PR!

**Comment by Shuckdubbie** [comment]
This pull is not valid. I will make another pull from the final code.

**Pull request closed without merging**